drivers/include/at86rf215.h

```diff
@@ -292,8 +292,9 @@ typedef struct at86rf215 {
  * @param[out] dev_09       sub-GHz device descriptor
  * @param[out] dev_24       2.4 GHz device descriptor
  * @param[in]  params       parameters for device initialization
+ * @param[in]  index        Index of the device in the parameter struct
```

Comment:
This documentation needs to be clarified. Currently it could be misunderstood to be the the index in `params`, which could be fatal when used that way. Maybe

```suggestion
 * @param[in]  index        index of @p params in a global parameter struct array.
```

There also needs to be documentation for the case if this does not come from a global struct array (use-case: user does not use auto-init and initializes the device "by hand").

Also, since when do structs have indices? 

(Also also please stick to the capitalization style of the rest of the function's doc)

Reply:
Ok, fixed the documentation.